src/array-utils.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We could do this with lodash (and other utils). However, since we're not currently using webpack, that would be non-ideal since we do not have tree-shaking.

@dbjorge any specific reasoning (other than timing) for adding webpack + lodash?

(not looking to block this PR on this but did want to point it out)

Reply:
We are already introducing lodash.

Reply:
Never mind, we talked about it offline; ignore.

Reply:
My mistake, the usage of lodash is a dev-dependency.